components/payloads/cu_ros_payloads/src/builtin.rs

```diff
@@ -0,0 +1,13 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct Header {
+    pub stamp: Time,
+    pub frame_id: std::string::String,
```

Comment:
Nit: if the string is short maybe use a short string like compact_str we use in cu29_runtime?